File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int mx=max(0,gain[0]);
        for(int i=0;i<gain.size()-1;i++){
            gain[i+1]+=gain[i];

            mx=max(mx,gain[i+1]);
        }
        return mx;
    }
};

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        int mx=Math.max(0,gain[0]);
        for(int i=0;i<gain.length-1;i++){
            gain[i+1]+=gain[i];

            mx=Math.max(mx,gain[i+1]);
        }
        return mx;
    }
}